test/support/fixtures/words_fixtures.ex

```diff
@@ -0,0 +1,22 @@
+defmodule BetterWords.WordsFixtures do
```

Comment:
This file is 100% generated. At some point we will move away from "fixtures" and look at "factories" but this is a fine pattern for us to get started with.